_schemaV2.graphql

```diff
@@ -8416,6 +8422,9 @@ type Viewer {
   artwork(
     # The slug or ID of the Artwork
     id: String!
+
+    # Include unlisted artwork or not
+    includeUnlisted: Boolean
```

Comment:
This is optional argument and hopefully not a breaking change to existing behavior.